src/test/java/uk/gov/hmcts/reform/amapi/controllers/AccessManagementControllerTest.java

```diff
@@ -194,4 +203,37 @@ public void testFilterResourceWithSecurityClassification() throws Exception {
             .andExpect(jsonPath("$.userSecurityClassification", is("PUBLIC")))
             .andExpect(jsonPath("$.resource.data.json", is("resource")));
     }
+
+    @Test
+    public void testReturnResourceAccessors() throws Exception {
+
+        String resourceType = "case";
+        String resourceName = "claim";
+        String resourceId = "0011";
+
+        ResourceAccessorsEnvelope resourceAccessorsEnvelope = ResourceAccessorsEnvelope.builder()
+            .resourceId(resourceId)
+            .explicitAccessors(Collections.singletonList(ResourceAccessor.builder()
+                .accessorId("5511")
+                .accessorType(USER)
+                .relationships(ImmutableSet.of("caseworker"))
+                .permissions(Collections.singletonMap(JsonPointer.valueOf(""), ImmutableSet.of(READ, UPDATE, CREATE)))
+                .build()))
+            .build();
+
+        Mockito.when(filterResourceService.returnResourceAccessors(resourceId, resourceName, resourceType))
+            .thenReturn(resourceAccessorsEnvelope);
+
+        this.mockMvc.perform(get("/api/resource/resourceType/" + resourceType + "/resourceName/"
+            + resourceName + "/resourceId/" + resourceId))
+            .andDo(print())
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.resourceId", is("0011")))
+            .andExpect(jsonPath("$.explicitAccess").exists())
+            .andExpect(jsonPath("$.explicitAccess[0].accessorId", is("5511")))
```

Comment:
there should be a test case for HTTP Status code - 404 when the search resource is not found

Reply:
👍  returns empty array as discussed

Reply:
Ben - there should be an API method for this test case as this is important scenario for a GET API request

Reply:
oh i guess we discussed this and agreed to add this to Serenity tests once they are ready!